Checks/PWR032/example.c

```diff
@@ -3,6 +3,11 @@
 
 #include <math.h>
 
-void example(float x) {
-  float res = sin(x);
+void example(const int N, const float *X1, const float *Y1, const float *X2,
```

Comment:
Just for clarity, we could add a brief comment describing the computation:

```c
// Euclidean distance between pairs of 2D points
```

WDYT?

Reply:
Yeah, I'll include that comment.